src/qt/coincontroldialog.cpp

```diff
@@ -78,18 +78,18 @@ CoinControlDialog::CoinControlDialog(QWidget *parent) :
     connect(clipboardLowOutputAction, SIGNAL(triggered()), this, SLOT(clipboardLowOutput()));
     connect(clipboardChangeAction, SIGNAL(triggered()), this, SLOT(clipboardChange()));
 
-    ui->labelCoinControlQuantity->addAction(clipboardQuantityAction);
-    ui->labelCoinControlAmount->addAction(clipboardAmountAction);
-    ui->labelCoinControlFee->addAction(clipboardFeeAction);
-    ui->labelCoinControlAfterFee->addAction(clipboardAfterFeeAction);
-    ui->labelCoinControlBytes->addAction(clipboardBytesAction);
-    ui->labelCoinControlPriority->addAction(clipboardPriorityAction);
-    ui->labelCoinControlLowOutput->addAction(clipboardLowOutputAction);
-    ui->labelCoinControlChange->addAction(clipboardChangeAction);
+    ui->coinControlQuantityLabel->addAction(clipboardQuantityAction);
+    ui->coinControlAmountLabel->addAction(clipboardAmountAction);
+    ui->coinControlFeeLabel->addAction(clipboardFeeAction);
+    ui->coinControlAfterFeeLabel->addAction(clipboardAfterFeeAction);
+    ui->coinControlBytesLabel->addAction(clipboardBytesAction);
+    ui->coinControlPriorityLabel->addAction(clipboardPriorityAction);
+    ui->coinControlLowOutputLabel->addAction(clipboardLowOutputAction);
+    ui->coinControlChangeLabel->addAction(clipboardChangeAction);
 
     // toggle tree/list mode
-    connect(ui->radioTreeMode, SIGNAL(toggled(bool)), this, SLOT(radioTreeMode(bool)));
-    connect(ui->radioListMode, SIGNAL(toggled(bool)), this, SLOT(radioListMode(bool)));
+    connect(ui->treeModeRadioButton, SIGNAL(toggled(bool)), this, SLOT(treeModeRadioButton(bool)));
```

Comment:
there is a critical error in this PR here.  I believe the SLOT should not have been changed